manifests/base/applicationset-controller/argocd-applicationset-controller-role.yaml

```diff
@@ -27,6 +27,8 @@ rules:
       - appprojects
     verbs:
       - get
+      - list
+      - watch
```

Comment:
i think it only complained about list permissions, why we need watch?

Reply:
See https://github.com/argoproj/argo-cd/issues/18829#issuecomment-2202098849

Reply:
Hm, okay, for me it was just list, weird